hw/ip/prim/data/prim_mubi_pkg.sv.tpl

```diff
@@ -0,0 +1,155 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// ------------------- W A R N I N G: A U T O - G E N E R A T E D   C O D E !! -------------------//
+// PLEASE DO NOT HAND-EDIT THIS FILE. IT HAS BEEN AUTO-GENERATED WITH THE FOLLOWING COMMAND:
+//
+//    hw/ip/prim/util/generate_prim_mubi_pkg.py hw/ip/prim/data/prim_mubi_pkg.sv.tpl > \
+//                                              hw/ip/prim/rtl/prim_mubi_pkg.sv
+//
+// This package defines common multibit signal types, active high and active low values and
+// the corresponding functions to test whether the values are set or not.
+
+package prim_mubi_pkg;
+
+% for n in range(1, n_max_nibbles + 1):
+<%
+  nbits = n * 4
+  hi_val = ''
+  lo_val = ''
+  for k in range(1,n+1):
+    hi_val = ('5' if (k % 2) else 'A') + hi_val
+    lo_val = ('A' if (k % 2) else '5') + lo_val
+%>\
+  //////////////////////////////////////////////
+  // ${nbits} Bit Multibit Type and Functions //
+  //////////////////////////////////////////////
+
+  parameter int MuBi${nbits}Width = ${nbits};
+  typedef enum logic [MuBi${nbits}Width-1:0] {
+    MuBi${nbits}Hi = ${nbits}'h${hi_val}, // enabled
+    MuBi${nbits}Lo = ${nbits}'h${lo_val}  // disabled
+  } mubi${nbits}_e;
+
+  // Set the multibit value to signal "enabled".
```

Comment:
I don't think this is quite the right name: this function doesn't set anything. Rather it returns the high value. Maybe something like `mubi${nbits}_hi_value`?

Reply:
sure I can do that. 

I know that these are very degenerate functions right now - I added them since it makes the assignments stick out in the code (for audit purposes), and in order to have an easy way to extend this in the future (e.g. by adding arguments to this function etc).